src/components/layout/NavBar.jsx

```diff
@@ -107,17 +107,21 @@ class NavBar extends Component {
         return this.canUserSeeComponent(item.name) ? navButton : null;
       });
 
+    // added for backward compatibility
+    // should always add homepageHref to components in portal config in the future
+    const homepageHref = config.homepageHref || components.homepageHref;
```

Comment:
if `config.homepageHref` is deprecated maybe we should use `const homepageHref = components.homepageHref || config.homepageHref;`

Reply:
ohh good call